src/Action/ExceptionAction.php

```diff
@@ -42,17 +42,20 @@
     private $serializer;
     private $errorFormats;
     private $exceptionToStatus;
+    private $debug;
 
     /**
      * @param SerializerInterface $serializer
      * @param array               $errorFormats      A list of enabled formats, the first one will be the default
      * @param array               $exceptionToStatus A list of exceptions mapped to their HTTP status code
+     * @param bool                $debug             The kernel debug parameter
      */
-    public function __construct(SerializerInterface $serializer, array $errorFormats, array $exceptionToStatus = [])
+    public function __construct(SerializerInterface $serializer, array $errorFormats, array $exceptionToStatus = [], bool $debug = false)
```

Comment:
Because this is a security issue and needs to be fixed in 2.0, I think we should maintain the debug flag here.